src/scenes/game-scene.ts

```diff
@@ -28,20 +35,24 @@ class GameScene extends Scene {
     // toggleDebug(this);
     this.input.keyboard?.on('keydown-CTRL', () => toggleDebug(this));
 
-    this.matter.world.setBounds();
+   
     this.matter.add.mouseSpring();
 
     const { create } = parallax(this);
     create();
-
-    this.ball = new Ball(this, cx, cy);
-
+    
+    this.map.create();
+    this.matter.world.setBounds(this.map.x, this.map.y, this.map.width, this.map.height);
+
+    let playerPos = this.map.spawners.player;
```

Comment:
![Screenshot from 2023-11-14 13-04-35](https://github.com/ScottyXIII/bigheaded/assets/4681349/047616ba-6eb2-4bf5-b22c-dbf1b4bfe6b8)

is the linting working for you?

this should be converted to const automatically when saving the file